api/schemas.py

```diff
@@ -1,47 +1,33 @@
-from marshmallow_sqlalchemy import SQLAlchemySchema, auto_field
-from .models import Category, Bot, User
+from marshmallow import Schema, fields
 
 
-class CategorySchema(SQLAlchemySchema):
-    class Meta:
-        model = Category
-        include_relationships = True
-        load_instance = True
+class BotSchema(Schema):
+    id = fields.Integer(dump_only=True)
```

Comment:
not consistent use of names: here you use full name and for category id it's alias Int